src/test/shared/credentials/loginManager.test.ts

```diff
@@ -102,7 +102,7 @@ describe('LoginManager', async function () {
 
         await loginManager.login({ passive: true, providerId: sampleCredentialsProviderId })
         assert.strictEqual(setCredentialsStub.callCount, 1, 'Expected awsContext setCredentials to be called once')
-        assert.strictEqual(recordAwsSetCredentialsSpy.calledOnce, false)
+        assert.strictEqual(recordAwsSetCredentialsSpy.calledOnce, true)
```

Comment:
The tests are less meaningful now, since `recordAwsSetCredentials` is _always_ called. We can add meaning by checking `calledOnceWith()`. 

```suggestion
        assert.strictEqual(recordAwsSetCredentialsSpy.calledOnceWith({
            credentialType: 'staticProfile',
            passive: true,
        }), true)

```